### DISCUSSION TITLE
sccache for windows arm64?

### DISCUSSION THREAD

**Discussion by hjyamauchi** [mozilla/sccache#2223] [Q&A] [answered]
Hi, does sccache support windows arm64 or plan to? there doesn't seem to be a windows arm64 build in the release. Thanks.

**Comment by Xuanwo** [comment]
It should be fine to add a windows arm64 build. Would you like to contribute such a workflow?

Add a new job here should work:

https://github.com/mozilla/sccache/blob/42db6f101101577ef32775bbb726ef93a92d7aec/.github/workflows/ci.yml#L136-L162

**Reply by hjyamauchi** [reply]
I'll see what I can do :)

**Reply by hjyamauchi** [reply]
@Xuanwo
Here's a PR: https://github.com/mozilla/sccache/pull/2243
I am hoping that this is as simple as this though this might involve cross-compiling :)
Please let me know if there's a way for me to test it.

**Reply by hjyamauchi** [reply]
@Xuanwo
The PR https://github.com/mozilla/sccache/pull/2243 has been merged!
Do you know when a new release that would include the windows arm64 binary may come?

**Reply by hjyamauchi** [reply]
Here's [the windows arm64 binary](https://github.com/mozilla/sccache/actions/runs/10288584504/artifacts/1786862538) from [the CI run](https://github.com/mozilla/sccache/actions/runs/10288584504).

**Reply by hjyamauchi** [reply, marked as answer]
It looks like we now have a windows arm64 binary in the latest release. https://github.com/mozilla/sccache/releases/tag/v0.8.2

Thanks!
